hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/authority/CertificateServer.java

```diff
@@ -112,6 +114,16 @@ X509Certificate getCertificate(String certSerialId)
    * framework.
    */
 
+  /**
+   * List certificates.
+   * @param type            - node type: OM/SCM/DN
+   * @param startSerialId   - start certificate serial id
+   * @param count           - max number of certificates returned in a batch
+   * @return
+   * @throws IOException
+   */
+  List<X509Certificate> listCertificate(HddsProtos.NodeType type,
```

Comment:
Nit: getCertificates may be a better name here
```suggestion
  List<X509Certificate> getCertificates(HddsProtos.NodeType type,
```

Reply:
Same as above.